src/Diagrams/Prelude.hs

```diff
@@ -132,8 +140,11 @@ module Diagrams.Prelude
          -- | For working with 'Active' (i.e. animated) things.
        , module Data.Active
 
-         -- | Essential Lens Combinators
-       , (&), (.~), (%~)
+         -- | This exports most of the lens module, excluding the
```

Comment:
Should be lens "package" not module.  Or maybe it should say "Control.Lens module".